.circleci/config.yml

```diff
@@ -0,0 +1,160 @@
+version: 2.1
+
+orbs:
+  aws-s3: circleci/aws-s3@1.0.11
+
+only_branches: &only_branches
+  filters:
+    branches:
+      ignore: master
+
+only_master: &only_master
+  filters:
+    branches:
+      only: master
+
+only_v_tags: &only_v_tags
+  filters:
+    tags:
+      only: /^v.*/
+    branches:
+      ignore: /.*/
+
+jobs:
+  build-ltops-and-test:
+    docker:
+      - image: circleci/golang:1.12
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependencies-{{ checksum "go.sum" }}
+      - run:
+          name: Install dependencies
+          command: |
+            export PATH=$PATH:$GOPATH/bin
+            mkdir -p $GOPATH/bin
+            make install
+      - save_cache:
+          key: dependencies-{{ checksum "go.sum" }}
+          paths:
+            - $GOPATH/pkg/mod
+      - run:
+          name: Build package
+          command: make package
+      - run:
+          name: Test
+          command: |
+            set +x
+            ltops --help
+      - persist_to_workspace:
+          root: .
+          paths:
+            - dist
+
+  deploy-branches:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - attach_workspace:
+          at: .
+      - aws-s3/sync:
+          from: dist/mattermost-load-test.tar.gz
```

Comment:
i think you need to add the original path

which I think is something like `mattermost-load-test/dist/...` can you please check?

Reply:
Thanks, updated.